### DISCUSSION TITLE
Pricing?

### DISCUSSION THREAD

**Discussion by argh-tom** [slackhq/nebula#1290] [Q&A] [answered]
Hello! I can't understand how nebula and the prices on [this page](https://www.defined.net/pricing/) are related. Do the desktop and mobile apps require a subscription to use it fully? 
If I set up the nebula server myself and don't use a subscription, will my use of these apps be limited to the free plan?

**Comment by nbrownus** [comment, marked as answer]
nebula is free to use and has no subscription model.

Defined Networking runs a service that makes it easier to deploy and maintain a nebula network.
